crates/common/src/compile.rs

```diff
@@ -341,11 +397,28 @@ pub fn deployed_contract_size<T: Artifact>(artifact: &T) -> Option<usize> {
     Some(size)
 }
 
+pub fn initcode_size<T: Artifact>(artifact: &T) -> Option<usize> {
```

Comment:
Since `initcode_size` is similar with `deployed_contract_size` let's refactor that one to `contract_size` and use for both runtime / init sizes, like

```diff
-/// Returns the size of the deployed contract
-pub fn deployed_contract_size<T: Artifact>(artifact: &T) -> Option<usize> {
-    let bytecode = artifact.get_deployed_bytecode_object()?;
+/// Returns the deployed or init size of contract.
+fn contract_size<T: Artifact>(artifact: &T, initcode: bool) -> Option<usize> {
+    let bytecode = if initcode {
+        artifact.get_bytecode_object()?
+    } else {
+        artifact.get_deployed_bytecode_object()?
+    };
     let size = match bytecode.as_ref() {

```